unlock-app/src/components/lock/ShowUnlessUserHasKeyToAnyLock.js

```diff
@@ -16,6 +17,7 @@ export const ShowUnlessUserHasKeyToAnyLock = ({
 
   // There is a valid key, but we shown the modal previously
   if (keys.length > 0 && !!modalShown) {
+    unlockPage()
```

Comment:
I actually have a PR coming that refactors all this because I think we need to use `lifecycle` functions here. Please bear with me ;)